utils/deploy.ts

```diff
@@ -0,0 +1,30 @@
+export function deployTruffleArtifacts(
```

Comment:
Nice I assume this is what we'll use in the machine to deploy the contracts?

Reply:
Something like this, but with a `provider` instead of a `deployer` akin to https://github.com/0xProject/0x-monorepo/blob/a21bfc5f2c9e7e8b660f9ea6671e1dfd229ab75d/packages/contract-wrappers/test/global_hooks.ts

`deployTruffleArtifacts` was mostly an attempt for me to figure out whether this worked at all